### PR TITLE
Added all research and class reports written for induction over the p…

### PR DESCRIPTION
**Pull request by isinha28**
Submitted all reports for classes and research papers from last 2 semesters and checked the format

# Summary of Changes

- Please indicate the classes you are modifying:
  - Cores, electives, math-science reqs
  - Research papers

# Checklist:

Make sure that your contributions:

- [Y] Are accurate and will not mislead the reader
- [Y] Do not encourage or cause academic integrity violations
- [Y] Do not offend professors or unfairly criticize the university

### CODE REVIEW

**Review by graysonmoyer22 — APPROVED** [review]

File: electives/18665.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (mikinty) [inline electives/18665.md:60]: Extra spaces

File: electives/18785.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (mikinty) [inline electives/18785.md:1]: Extra space

File: electives/24104.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+ |
+
+|
+
+Homeworks
+
+ |
+
+2
+
```
Comment (mikinty) [inline electives/24104.md:20]: ??? what's going on here?

File: electives/24104.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+ |
+|
+
+Final Project
+
+ |
+
+3
```
Comment (mikinty) [inline electives/24104.md:73]: Same here? Also is this copied twice?

File: electives/80180.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+ |
+
+Difficulty
+
+ |
+|
+
+Homework
+
+ |
```
Comment (mikinty) [inline electives/80180.md:16]: This is also really funky?

**Pull request merged**